frontend/src/pages/LogsPage/SearchForm/SearchForm.tsx

```diff
@@ -148,7 +145,7 @@ const SearchForm = ({
 					{!hideDatePicker && (
 						<PreviousDateRangePicker
 							emphasis="low"
-							selectedDates={selectedDates}
+							selectedDates={[startDate, endDate]}
```

Comment:
Think this is a pre-existing issue, but may be making it more obvious that we set the end date when reloading the page, or fetching more results

https://github.com/highlight/highlight/assets/17744174/94a0df95-cd8e-4906-946f-4da3480e3b70

Reply:
fixed in https://github.com/highlight/highlight/commit/b2c888243c11c97daa64e53fe06959eb21343b70